lib/middleware/getListOfLicences.js

```diff
@@ -1,32 +1,31 @@
 const accessLicenceClient = require('kat-client-proxies').accessLicenceClient;
 
 module.exports = (req) => {
-    return new Promise(resolve => {
+    return new Promise((resolve, reject) => {
         if (req.currentUser && req.currentUser.uuid) {
             if (Array.isArray(req.listOfLicences)) {
                 return resolve(req.listOfLicences);
             }
 
-            return resolve(
-                accessLicenceClient.getLicences({adminuserid: req.currentUser.uuid})
-                    .then(response => {
-                        if (Array.isArray(response)) {
-                            req.listOfLicences = response
-                                .map(item => ({
-                                    licenceId: item.id,
-                                    creationDate: item.creationDateTime,
-                                    status: item.status,
-                                    contractId: item.links[0].id,
-                                    product: item.products[0].name
-                                }))
-                                .filter(item => item.status === 'active');// remove the licences that are not active
+            return accessLicenceClient.getLicences({adminuserid: req.currentUser.uuid})
+                .then(response => {
+                    if (Array.isArray(response)) {
+                        req.listOfLicences = response
```

Comment:
I guess you are right and we might want to move this file elsewhere as it's not really a middleware. Will do it in our next update.